bedrock/settings/base.py

```diff
@@ -233,10 +235,10 @@ def JINJA_CONFIG():
     }
 
 
-MEDIA_URL = '/user-media/'
-MEDIA_ROOT = path('media')
-STATIC_URL = '/media/'
-STATIC_ROOT = path('static')
+MEDIA_URL = config('MEDIA_URL', default='/user-media/')
```

Comment:
Agreed it's unlikely, this is more future proofing than strictly necessary, but I came to the same conclusion that it couldn't hurt :)